buildSrc/src/test/java/org/elasticsearch/gradle/BuildExamplePluginsIT.java

```diff
@@ -86,6 +86,7 @@ public void testCurrentExamplePlugin() throws IOException {
 
         GradleRunner.create()
             .withProjectDir(tmpDir.getRoot())
+            .withTestKitDir(tmpDir.newFolder("testkit"))
```

Comment:
This shouldn't be necessary. We are already running each of these test cases in a new temporary directory. By default TestKit is going to create a `.gradle-test-kit` directory under that folder. Really all this is doing is renaming that folder `testkit` instead. If the example plugin tests are failing, it's not due to testkit directory reuse.